src/blocks/ui/components/block-ab-test.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { compactMetric, getLetter, getLift } from '../../../utils';
+import { defaultVariantAnalytics } from '../../data/shapes';
+
+import Cards from './cards';
+import Timeline from './timeline';
+import Variants from './variants';
+
+const { useSelect } = wp.data;
+const { __, sprintf } = wp.i18n;
+
+/**
+ * Helper to get a variants title.
+ *
+ * @param {string} title The defined title if available.
+ * @param {number} id The variant ID or index.
+ * @returns {string} The variant title.
+ */
+const getABVariantTitle = ( title, id ) => title || sprintf( __( 'Variant %s', 'altis-analytics' ), getLetter( id ) );
+
+/**
+ * Personalized Content Block Analytics component.
+ *
+ * @param {object} props The component props.
+ * @param {object} props.block The block post data.
+ * @param {string} props.clientId The block client ID.
+ * @returns {React.ReactNode} The block view component.
+ */
+const BlockABTest = ( {
```

Comment:
Does it make sense to namespace this somehow ?

Reply:
Yeah it's only really namespaced by the directory it's in right now 🤔  I'll have a think.